doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -1079,3 +1059,14 @@ For more details on how to pass callbacks to C libraries, see this [blog post](h
 
 For direct C++ interfacing, see the [Cxx](https://github.com/Keno/Cxx.jl) package. For tools to create C++
 bindings, see the [CxxWrap](https://github.com/JuliaInterop/CxxWrap.jl) package.
+
+
+
+[^1]: Non-library function calls in both C and Julia can be inlined and thus may have
+    even less overhead than calls to shared library functions. When both libraries and executables are
+    generated by LLVM, it is possible to perform whole-program optimizations that can even optimize
```

Comment:
```suggestion
```

Reply:
just out of curiosity, is this impossible for now?

Reply:
no, some packages do it, it's just largely nonsense and irrelevant for the manual